routes/web.php

```diff
@@ -30,3 +31,6 @@
 Route::get('/login/google/callback', [LoginController::class, 'handleGoogleCallback']);
 
 Route::get('logout', [LoginController::class, 'logout'])->name('logout');
+Route::get('wish-list', [WishListController::class, 'index'])->name('wishList.form')->middleware('auth');
+Route::post('create/wish-list', [WishListController::class, 'create'])->name('create.wishList');
```

Comment:
this route also should be available for the authenticated user.
I'd suggest to use [route group](https://laravel.com/docs/10.x/routing#route-group-middleware)

Reply:
and in the ticket it has been mentioned we need a plus button to add the new wish.
not a new create wish page